client/src/pages/schema/IndexTypeElement.tsx

```diff
@@ -86,18 +92,15 @@ const IndexTypeElement: FC<{
   const DeleteIcon = icons.delete;
 
   const fetchStatus = useCallback(async () => {
-    if (data._indexType !== '') {
+    // prevent delete index trigger fetching index status
+    if (data._indexType !== '' && status !== IndexState.Delete) {
```

Comment:
Default state here ?

Reply:
`indexType` is not index status. If no index, it will be empty string, otherwise it should return index type name like `FLAT`